scripts/create_newcase

```diff
@@ -156,6 +156,9 @@ def parse_command_line(args, cimeroot, description):
 
     parser.add_argument("--case-group", help="Add this case to a case group")
 
+    parser.add_argument("--ngpus_per_node", default=0, type=int,
```

Comment:
We should stick with standards-compliant option names, `--ngpus-per-node` instead of `--ngpus_per_node`

Reply:
Thanks for the comment. Will update the option name soon.